events/kwoc.md

```diff
@@ -11,13 +11,18 @@
 - The following things should be ready (or updated) :-
 
   - Website with registration feature tested internally.
+  - Last year's stats on the website.
```

Comment:
Maybe we could add updating testimonials here?

Reply:
umm can be a good plan, but for that we need to have testimonials. Do we have previous year's testimonials?

Reply:
"Update" can mean everything from checking for link rot to adding new testimonials if we have any

Reply:
@proffapt We sent a survey form to participants right? Can use that

Reply:
yea, ok makes sense

Reply:
Done